File: C Applications/Mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (1 > n || 8 < n);
    for (int i = 1; i <= n; i++)
    {
        for (int l = n; l > i; l--)
        {
            printf(" ");
        }
        
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        for (int k = 0; k < 2; k++)
        {
            printf(" ");
        }
        for (int h = 0; h < i; h++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Python Applications/Mario-more/mario.py
import cs50
while True:
    n = cs50.get_int("Height: ")
    if 1 <= n <= 8:
        for i in range(1, n+1):
            for l in range(n, i, -1):
                print(" ", end="")
            for j in range(0, i):
                print("#", end="")
            for k in range(2):
                print(" ", end="")
            for h in range(0, i):
                print("#", end="")
            print()
        break